mustang_codegen/lib/src/state_generator/screen_state_generator.dart

```diff
@@ -198,7 +198,11 @@ class ScreenStateGenerator extends Generator {
             element: element);
       }
 
-      if (element.type.element != null && !_hasAppModelAnnotation(element)) {
+      if (element.type.element != null &&
+          !element.type.element!.declaration
+              .toString()
+              .contains('implements Built') &&
+          !_hasAppModelAnnotation(element)) {
```

Comment:
in our apps, we have models from both app repo and common repo.

Reply:
this condition blocks generation when models are generated ones instead of user written ones

Reply:
any model that doesn't have `$` throws error
![Screenshot 2022-02-26 at 8 14 09 PM](https://user-images.githubusercontent.com/26526504/155847243-7b7eb584-d0ab-4fc4-b163-41c5296ac34a.png)